pocs/dome/astrohaven.py

```diff
@@ -157,7 +159,8 @@ def _full_move(self, send, target_feedback):
         # In other words, we'll try to read status, but if it isn't available,
         # we'll just send another command.
         saved_timeout = self.serial.ser.timeout
-        self.serial.ser.timeout = 0.2
+        self.serial.ser.timeout = max(0.0000001, AstrohavenDome.MOVE_LISTEN_TIMEOUT)
```

Comment:
Just to guarantee timeout is never 0?

Reply:
Yes, though I now see that 0 does not imply forever. None is used for that. 0 means non-blocking (i.e. read as much as available AND requested, but don't wait if requested amount/terminator is not there).